static/js/protein/chart.ts

```diff
@@ -242,34 +258,77 @@ const LINK_STYLE = {
   },
 };
 
-// https://stackoverflow.com/a/69610045
-function brighten(): void {
-  // brighten object under cursor
-  d3.select(this).style("filter", `brightness(${BRIGHTEN_PERCENTAGE})`);
-}
-
-function unbrighten(): void {
-  // unbrighten object under cursor
-  d3.select(this).style("filter", "brightness(100%)");
+/**
+ * When mouse first enters element specified by given id, brighten it and update/display the global tooltip.
+ */
+function onMouseOver(
+  elementID: string,
+  toolTipText: string,
+  brightenPercentage: string = DEFAULT_BRIGHTEN_PERCENTAGE
+): void {
+  // brighten element: https://stackoverflow.com/a/69610045
+  d3.select(`#${elementID}`).style(
+    "filter",
```

Comment:
Thank you so much for adding the brightness component. It really enhances the aesthetics of the chart. I was wondering if we'd wanna add the style elements to the protein.scss file ?

Reply:
usually we do want to add styling to the scss file, however, since the brightenPercentage is an argument that gets passed depending on the chart and you can't pass arguments to scss, I think it's easier to set the style here.

Reply:
Oh! Good point! I'm not sure about which is more kosher - so on the one hand the brightness percentage is a variable which motivates handling this in `chart.ts`, but on the other hand if the number of brightness values we want to support is small we could just enumerate all of them in a few CSS classes.  @chejennifer thoughts on this?

Reply:
Oops sorry for not refreshing/thanks for answering my question before I asked it!